OneSignalSDK/build.gradle

```diff
@@ -1,36 +1,59 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.72'
+
+    ext {
```

Comment:
In this root `build.gradle` I believe we should only define `ext` versions we want to share between more than 1 `build.gradle` file. Example `jUnitVersion` doesn't need to be her since it will only ever be used in the `unittest` project.

Reply:
Ok I divided on different ext, but kept the definition of the variables just for coding convention

Reply:
Let me know what you think

Reply:
I like it, looks good to me!